aws/ecs_cluster.go

```diff
@@ -0,0 +1,132 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+const firstSeenTagKey = "cloud-nuke-first-seen"
```

Comment:
NIT: don't we already have a variable like this? Could we use the same/consistent var everywhere?

Reply:
Good point - we do use it just in one other place - the `eip.go` module. I've added a `globals.go` file with the constant declared there